internal/kafka/internal/services/kafka.go

```diff
@@ -1036,6 +1039,10 @@ func buildManagedKafkaCR(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 		"bf2.org/kafkaInstanceProfileQuotaConsumed": strconv.Itoa(k.QuotaConsumed),
 		"bf2.org/kafkaInstanceProfileType":          kafkaRequest.InstanceType,
 	}
+
+	suspendedStatuses := []string{constants.KafkaRequestStatusSuspending.String(), constants.KafkaRequestStatusSuspended.String()}
+	labels["bf2.org/suspended"] = fmt.Sprintf("%t", arrays.Contains(suspendedStatuses, kafkaRequest.Status))
```

Comment:
can we add an integration test on the newly added label? There is this existing test that we could enrich https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/c030410217731ebb6c3e6dce12ddcd83c1766beb/internal/kafka/test/integration/data_plane_endpoints_test.go#L223 to test for the case the label is `false` (all the other statues - existing ones) and for when the label is `true`.

Reply:
yes. I am also adding some more unit tests right now

Reply:
Thank you.